pages/common/hr.md

```diff
@@ -0,0 +1,15 @@
+# hr
+
+> Print a horizontal rule in the terminal.
+
+- Print a horizontal rule:
+
+`hr`
+
+- Print a horizontal rule with a custom string:
+
+`hr {{string}}`
+
+- Print a multiline horizontal rule:
+
+`hr {{string}} {{string}} {{string...}}`
```

Comment:
Remove the ... from the last string.

Reply:
You could always do `string_a`, `string_b`, and `string_c` instead.

Reply:
I'll go with the latter to indicate they are different.
I wanted to indicate that there is no limit to the number of strings, but since tldr is aimed at examples more than a full man page, I'm ok with not explicitly saying that.